src/main/java/com/limechain/babe/state/EpochState.java

```diff
@@ -46,4 +47,12 @@ public void updateNextEpochBlockConfig(byte[] message) {
     public BigInteger getCurrentSlotNumber() {
         return BigInteger.valueOf(Instant.now().toEpochMilli()).divide(slotDuration);
     }
+
+    public BigInteger getCurrentEpochStartSlotNumer() {
+        return getCurrentEpochIndex().multiply(epochLength).add(genesisSlotNumber);
+    }
+
+    public BigInteger getCurrentEpochIndex() {
+        return getCurrentSlotNumber().subtract(genesisSlotNumber).divide(epochLength);
```

Comment:
Maybe add some kind of reference/comment regarding the formula used. This could also be done for methods above.